addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientListScreen.js

```diff
@@ -150,12 +170,28 @@ odoo.define('point_of_sale.ClientListScreen', function(require) {
             };
             this.render();
         }
-        async saveChanges(event) {
+        async saveChanges() {
             try {
+                let processedChanges = {};
+                for (let [key, value] of Object.entries(this.state.editModeProps.changes)) {
+                    if (this.intFields.includes(key)) {
+                        processedChanges[key] = parseInt(value) || false;
+                    } else {
+                        processedChanges[key] = value;
+                    }
+                }
+                if ((!this.state.editModeProps.partner.name && !processedChanges.name) ||
+                    processedChanges.name === '') {
```

Comment:
If it's okay with you, write the condition in one line.

Reply:
sure!